src/FunctionInfo.ts

```diff
@@ -28,13 +30,21 @@ export class FunctionInfo {
       Object.keys(bindings)
         .filter(name => bindings[name].direction !== rpc.BindingInfo.Direction.in)
         .forEach(name => {
-          if (bindings[name].type === 'http') {
+          let type = bindings[name].type;
+          if (type && type.toLowerCase() === 'http') {
             this.httpOutputName = name;
             this.outputBindings[name] = Object.assign(bindings[name], { converter: toRpcHttp });
           } else {
             this.outputBindings[name] = Object.assign(bindings[name], { converter: toTypedData });
           }
         });
+      
+      this.isHttp = Object.keys(bindings)
+        .filter(name => {
+          let type = bindings[name].type;
+          return type && type.toLowerCase() == 'httptrigger';
+        })
+        .length > 0;

```

Comment:
Is it ever valid to have 2+ bindings with trigger? I'm assuming somewhere in the pipeline we are already handling this invalid case.

Reply:
I don't think it's valid to have 2+ binding with trigger - assuming that is handled on the host end! I think it's ok to say > 0, although potentially less precise.